.prettierrc.json

```diff
@@ -1 +1,4 @@
-{}
+{
+  "trailingComma": "all",
```

Comment:
🔶 I think we should stick to the default here too:

https://prettier.io/docs/en/options.html#trailing-commas

The default puts trailing commas pretty much everywhere as-is. The difference in using an explicit `all` here is that it _also_ adds trailing commas in unusual places which aren't 100% standard.

Reply:
Fair. Downgraded to 🟢